configuration/pegasus.properties

```
@@ -5,5 +5,8 @@ enable_perf_counter = false
 perf_counter_tags = cluster=onebox,app=unit_test
 push_counter_interval_secs = 10
 meta_query_timeout = 5000
-service_name = ""
-service_fqdn = ""
+auth_protocol =
+service_name =
+service_fqdn =
+keytab =
+principal =
```

```suggestion
kerberos_service_name =
kerberos_service_fqdn =
kerberos_keytab =
kerberos_principal =
```

"kerberos_xxx"  helps users with bare knowledge of authentication quickly understand the meaning of these configs.
I don't even know what "principal" means actually if given no context of authentication.